common.blocks/control/control.js

```diff
@@ -106,7 +106,9 @@ provide(BEMDOM.decl(this.name, /** @lends control.prototype */{
 }, /** @lends control */{
     live : function() {
         this
-            .liveBindTo('control', 'focusin', this.prototype._onFocus)
+            .liveBindTo('control', 'focusin', function() {
```

Comment:
Because I don't want to increase a mess with managing focus state which we've already have.